immich-app/immich
# refactor(server): upload config

Tested Scenarios
- Upload profile picture from mobile app
- Upload file via drag and drop
- Upload file via upload button
- Upload duplicate via drag and drop
- Upload duplicate via upload button

Not Tested:
- XMP
- Live Photos

The code/logic should be the same, it has just been restructured in such a way that the logic (and tests) now exists in `AssetService`.
